src/components/LoginButton/LoginButtonIcon.js

```diff
@@ -26,6 +29,7 @@ LoginButtonIcon.propTypes = {
     title: PropTypes.string.isRequired,
     imageSrc: PropTypes.string.isRequired,
     href: PropTypes.string,
+    appearance: PropTypes.oneOf(LOGO_APPEARANCES).isRequired,
```

Comment:
@jiromaykin 
defined the possible values in a constant and used that for a more specific prop type - this matches the possible enum values from the API spec